remind.py

```diff
@@ -41,30 +41,42 @@ def run_remind(config, blockers, server, header):
 			# if jeeves was unable to collect any good jenkins API info, skip job
 			if jenkins_api_info:
 
-				# only care about jobs with UNSTABLE or FAILURE status
-				if jenkins_api_info['lcb_result'] == "UNSTABLE" or jenkins_api_info['lcb_result'] == "FAILURE":
+				# only care about jobs without SUCCESS status
+				if jenkins_api_info['lcb_result'] != "SUCCESS":
```

Comment:
I would recommend in future iterations tracking this state as a constant variable.